lgc/patch/MeshTaskShader.cpp

```diff
@@ -55,6 +61,201 @@ MeshTaskShader::~MeshTaskShader() {
   m_pipelineSysValues.clear();
 }
 
+// =====================================================================================================================
+// Layout mesh shader LDS if 'ldsLayout' is specified and calculate the required total LDS size (in dwords).
+//
+// @param pipelineState : Pipeline state
+// @param entryPoint : Entry-point of mesh shader
+// @param ldsLayout : Mesh shader LDS layout (could be null)
+unsigned MeshTaskShader::layoutMeshShaderLds(PipelineState *pipelineState, Function *entryPoint,
+                                             MeshLdsLayout *ldsLayout) {
+  if (!pipelineState->hasShaderStage(ShaderStageMesh))
+    return 0; // Mesh shader absent
```

Comment:
I don't add a assert here because this function is called when we just have a task shader and don't have a mesh shader. Of course, this is invalid in the spec because task shader is optional while mesh shader is required. We support this to compile a single task shader in standalone compiler.

Reply:
Ah, gotcha. It might help to add this to the comment, e.g. "Mesh shader absent (e.g. standalone compile of a single task shader)"

Reply:
Sure. Will modify.